Tests/PSGetInstallScript.Tests.ps1

```diff
@@ -121,6 +126,242 @@ function SuiteCleanup {
     }
 }
 
+
+
```

Comment:
Sure, I can do that.  Is there criteria on how to choose the tag?  I understand that BVT are most critical, then P1, then P2, but I don't fully understand where the line is between them.

---
In reply to: [132746237](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r132746237) [](ancestors = 132746237)